src/Codeception/Module/Symfony.php

```diff
@@ -22,6 +22,10 @@
  *
  * * Maintainer: **raistlin**
  * * Stability: **stable**
+ * 
+ * ## Compatibility
+ *
+ * This module is compatible with all other Codeception modules, except WebDriver and the modules for other PHP frameworks.
```

Comment:
See https://github.com/Codeception/Codeception/pull/4266/commits/903222a4e8dc3acccaa53dfdf2a0ba1606e018b5

More questions :-)

4. `var_path`: This is only required for Symfony 3 directory structure? Which path is to be used? Current documentation ("where bootstrap.php.cache is") is ambiguous, since I have multiple `bootstrap.php.cache`s. Best guess (i.e.) suggested value should probably be `var`, not `app`, right? I'm wondering, cause I have `app` in my config (Symfony 3 directory structure), and it still works.
4. What does `environment: 'local_test'` mean?
4. I omitted this link: http://lakion.com/blog/how-did-we-speed-up-sylius-behat-suite-with-blackfire  What's the relation to this module? Does the module use Blackfire?
4. Is it true for all parts that they are to be loaded in `acceptance.suite.yml`?
4. When loading a Symfony part, the other Symfony parameters (app_path, var_path, etc.) are not required? Or is `funcitonal.suite.yml` always required and they are taken from there?
4. `part: orm` only makes sense in combination with Doctrine? Or it there anything like a Symfony ORM by itself?
4. When loading a Symfony Part, it's not just *possible* but *mandatory* to load one of [PhpBrowser, WebDriver]?

Please merge what I have so far (if it doesn't contain a severe error) - I need to see a preview of what I already have..... :-)